### PR TITLE
fix: replace aio links -> adev links

### PR DESCRIPTION
**Pull request by celestius0**
## PR Checklist

Please check to confirm your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Various links throughout the codebase are still referencing angular.io 

Issue Number: N/A

## What is the new behavior?

Updated as many links as possible to point to the new locations on angular.dev. Specifically the following items were addressed:
- Update various guide and misc angular.io links
- Update all tutorial links
- Update all schematics links
- Update all security links
- Update all workspace config links
- Update all docs links
- Update all quick start links
- Update all blog links
- Update all cli links
- Update all optimization configuration links
- Update all browser support links
- Update all license file links

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
A few links could not be updated for the following reasons:

- `material.angular.io` these still appear to be valid for now
- `angular.io/config/tsconfig` there does not appear to be an equivalent resource on the angular.dev site

### CODE REVIEW

**Comment by davidlj95** [comment]
About tsconfig, made a proposal here #27714

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
Can you please update the commit messages as per guidelines https://github.com/angular/angular-cli/blob/main/CONTRIBUTING.md#commit

In this case using a `refactor` type seems more suitable.

**Comment by JeanMeche** [comment]
Can you please drop that merge commit and rebase on the main branch ? Thank you.

**Comment by celestius0** [comment]
> Can you please drop that merge commit and rebase on the main branch ? Thank you.

Yeah, sorry totally botched my rebase like a noob 😅. Should be good to go now but lemmie know if you see anything wonky.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ members of the project's leadership.
 
 ### Appeal
 
-If you are banned you may contest the decision. To do so email conduct@angular.io with the subject line "Repeal Ban for {{your name here}}" and body with the responses to the following:
+If you are banned you may contest the decision. To do so email conduct@angular.dev with the subject line "Repeal Ban for {{your name here}}" and body with the responses to the following:
```
Comment (alan-agius4) [inline CODE_OF_CONDUCT.md:70]: @josephperrott, does this email exist?

Comment (josephperrott) [reply CODE_OF_CONDUCT.md:70]: It does not

Comment (celestius0) [reply CODE_OF_CONDUCT.md:70]: Oop, okie dokie lemmie revert that back real quick and it can be addressed once it gets created.

**Review by alan-agius4 — APPROVED** [review]
LGTM, thanks

**Comment by alan-agius4** [comment]
The changes were merged into the following branches: main, 18.0.x

**Pull request merged**